src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy

```diff
@@ -45,4 +46,16 @@ class GroovyDummyServiceTest extends Specification {
         then:
         2 * dummyRepository.getDummyById(dummyId)
     }
+
+    def "should not deactivate dummy with negative id"() {

```

Comment:
Could you also extract the test to other file? I tried to keep the sample tests files as small and descriptive as possible.